src/discord/CommandHandler.js

```diff
@@ -30,7 +30,7 @@ class CommandHandler {
       return false
     }
 
-    if ((command.name != 'help' && !this.isCommander(message.member)) || (command.name == 'override' && !this.isOwner(message.author))) {
+    if ((command.name != 'help' && !this.isCommander(message.member)) && (command.name == 'override' && !this.isOwner(message.member))) {
```

Comment:
In this case the command will only be rejected if both:
- The user does not have the commander role and the command is not `help`
- The command is `override` and the user is not an owner

This means that anyone can run commands like `!mute` as the right side of the expression will only evaluate to true if the command name is `override`.